src/flexalloc_mm.c

```diff
@@ -26,6 +26,37 @@ fla_geo_zoned(const struct fla_geo *geo)
   return geo->type == XNVME_GEO_ZONED;
 }
 
+static int
+fla_dev_sanity_check(struct xnvme_dev const * dev, struct xnvme_dev const *md_dev)
```

Comment:
I wonder if flexalloc_mm is the proper place for this code. I think we should have a todo where we review flexalloc_mm and pull out functionality that is not core, possibly rename the file.

Reply:
Good idea. I'll set it up as a task to put MM on a diet :)

Reply:
Created this https://github.com/OpenMPDK/FlexAlloc/issues/53